src/cyclonedx/ui/CycloneDXGeneratorUi.php

```diff
@@ -0,0 +1,176 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2023 Sushant Kumar <sushantmishra02102002@gmail.com>
+
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+namespace Fossology\CycloneDX\UI;
+
+use Exception;
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\FolderDao;
+use Fossology\Lib\Dao\UploadDao;
+use Fossology\Lib\Data\Upload\Upload;
+use Fossology\Lib\Plugin\DefaultPlugin;
+use Symfony\Component\HttpFoundation\Request;
+
+class CycloneDXGeneratorUi extends DefaultPlugin
+{
+  const NAME = 'ui_cyclonedx';
+  const DEFAULT_OUTPUT_FORMAT = "cyclonedx";
```

Comment:
If updating output format in `cyclonedx.php`, please remember to update it here as well.